collector/kernel/cgroup_handler_test.cc

```diff
@@ -0,0 +1,456 @@
+      "MergedDir": "/var/lib/docker/overlay2/925ab32b43b5e0b27aae80af7cf4edfe129b79e56aca5c41337094eb95b83fea/merged",
+      "UpperDir": "/var/lib/docker/overlay2/925ab32b43b5e0b27aae80af7cf4edfe129b79e56aca5c41337094eb95b83fea/diff",
+      "WorkDir": "/var/lib/docker/overlay2/925ab32b43b5e0b27aae80af7cf4edfe129b79e56aca5c41337094eb95b83fea/work"
+    },
+    "Name": "overlay2"
+  },
+  "Mounts": [
+    {
+      "Type": "bind",
+      "Source": "/etc/flowmill/dev.pub",
+      "Destination": "/etc/flowmill/authz-token-public.pem",
+      "Mode": "",
+      "RW": true,
+      "Propagation": "rprivate"
+    },
+    {
+      "Type": "bind",
+      "Source": "/home/vagrant/src",
+      "Destination": "/root/src",
+      "Mode": "",
+      "RW": true,
+      "Propagation": "rprivate"
+    },
+    {
+      "Type": "bind",
+      "Source": "/home/vagrant/out",
+      "Destination": "/root/out",
+      "Mode": "",
+      "RW": true,
+      "Propagation": "rprivate"
+    }
+  ],
+  "Config": {
+    "Hostname": "ad87fc49c138",
+    "Domainname": "",
+    "User": "",
+    "AttachStdin": false,
+    "AttachStdout": true,
+    "AttachStderr": true,
+    "ExposedPorts": {
+      "7000/tcp": {},
+      "7001/tcp": {},
+      "7010/tcp": {},
+      "8000/tcp": {}
+    },
+    "Tty": true,
+    "OpenStdin": false,
+    "StdinOnce": false,
+    "Env": [
+      "FLOWMILL_TENANT_ID=100001",
+      "FLOWMILL_RUN_UNDER_GDB=",
+      "FLOWMILL_INGEST_ENABLE_NO_AUTH=true",
+      "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
+      "OPENSSL_CONF=/etc/openssl/openssl.cnf",
+      "GEOIP_PATH=/usr/share/GeoIP"
+    ],
+    "Cmd": [
+      "--port=8000",
+      "--internal-prom=0.0.0.0:7000",
+      "--prom=0.0.0.0:7001",
+      "--prom-queue-count=1",
+      "--ingest_shard_count=1",
+      "--matching_shard_count=1",
+      "--aggregation_shard_count=1",
+      "--enable-aws-enrichment",
+      "--log-console",
+      "--debug"
+    ],
+    "Image": "localhost:5000/pipeline-server",
+    "Volumes": null,
+    "WorkingDir": "",
+    "Entrypoint": [
+      "/srv/entrypoint.sh"
+    ],
+    "OnBuild": null,
+    "Labels": {
+      "org.label-schema.description": "Flowtune telemetry server",
+      "org.label-schema.name": "flowtune/flowtune-server",
+      "org.label-schema.schema-version": "1.0",
+      "org.label-schema.usage": "./README.md",
+      "org.label-schema.vcs-url": "https://review.prod.flowtune.io/deploy/flowtune-server-docker/",
+      "org.label-schema.vendor": "Flowmill, Inc"
+    }
```

Comment:
This test currently only validates these, but could be extended to validate more.